dev/io.openliberty.pages.3.1.internal.factories/bnd.bnd

```diff
@@ -0,0 +1,105 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Open Liberty Pages 3.1 Factories
+Bundle-SymbolicName: io.openliberty.pages.3.1.internal.factories
+Bundle-Description: WAS Pages 3.1, version ${bVersion}
+
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
+
+-dsannotations: com.ibm.ws.jsp23.webcontainerext.JspVersionFactoryImpl, \
+com.ibm.ws.jsp23.webcontainerext.ws.PrepareJspHelper23Factory, \
+com.ibm.ws.jsp23.translator.visitor.validator.ElValidatorExtFactoryImpl, \
+com.ibm.ws.jsp23.translator.visitor.generator.GeneratorUtilsExtFactoryImpl
+
+# Note these factories were copied from the com.ibm.ws.jsp.2.3 bundle
+
+Export-Package: \
+	com.ibm.ws.jsp.speclevel, \
+	com.ibm.ws.jsp23.webcontainerext;version="2.3", \
```

Comment:
A 2.3 version seems odd, lets discuss. Also the naming of the pages if in the 3.1 bundle is also odd to include jsp2.3.

Reply:
Oops. I will fix this.  I think I'll keep the com.ibm.ws. prefix  ( as I had issues when I tried to switch it to io.openliberty ), but changes jsp23 -> jsp31 and the version to 3.1